activity_browser/layouts/tabs/activity.py

```diff
@@ -72,6 +81,8 @@ class ActivityTab(QtWidgets.QWidget):
     The final table of this tab lists these 'Downstream Consumers'
     """
 
+    open_table_toggles = []
```

Comment:
I think it'd be better to include this in the `__init__` and call `open_table_toggles` from `self` instead of `ActivityTab`. We do sometimes create variables before init, but we only do that for static variables that don't change at runtime, like column names for a table etc. -do note that this may not be entirely consistent, and is something we do in AB, not a general python thing-.

Reply:
I'll put this in the ActivitiesTab and link it to the new instances so it can be shared. It's best to keep these things consistent

Reply:
Hmm, in this case I'm struggling to find a way to execute this that doesn't involve the static variable. While consistency would be good, the problem is that to store the list variable in the ActivitiesTab to store the hide/show status of the different flow tables, requires allowing the tables to modify the variable in ActivitiesTab. (ActivitiesTab being the class that contains the different ActivityTabs).